com.unity.render-pipelines.high-definition/Runtime/Lighting/AtmosphericScattering/AtmosphericScattering.hlsl

```diff
@@ -115,7 +115,7 @@ void EvaluatePbrAtmosphere(float3 worldSpaceCameraPos, float3 V, float distAlong
         // We may have swapped X and Y.
         float2 ch = abs(ch0 - ch1);
 
-        float3 optDepth = ch.x * H.x * _AirSeaLevelExtinction
+        float3 optDepth = ch.x * H.x * _AirSeaLevelExtinction.xyz
```

Comment:
Should be .rgb to avoid confusion.

Reply:
I did this to match the changes that went into master.  I'm not sure why .xyz is unclear (happy to make changes, just want to understand).

Reply:
It's a "color".

Reply:
Ah, I explicitly made this .xyz in the 'master' version of the PR because everything around it was .x or .y (same with the other instance in PhysicallyBasedSkyCommon.hlsl).  I assumed if it were colors, we would be using .r or .g.